src/Database/LSMTree/Common.hs

```diff
@@ -220,21 +220,23 @@ listSnapshots (Internal.Session' sesh) = Internal.listSnapshots sesh
 --
 -- Though blob references are handle-like, they /do not/ keep files open. As
 -- such, when a blob reference is returned by a lookup, modifying the
--- corresponding table handle (or session) /may/ cause the blob reference to be
--- invalidated (i.e.., the blob has gone missing because the blob file was
+-- corresponding table handle, cursor, or session /may/ cause the blob reference
```

Comment:
I don't follow. Cursors cannot be modified.

Reply:
Technically, closing cursor counts as modification, that's also why session close is listed as something that can invalidate a blob reference